packages/components/psammead-media-player/README.md

```diff
@@ -53,14 +69,26 @@ The `height` and `width` props are required to be provided in advance by AMP so
 ```js
 import { CanonicalMediaPlayer } from '@bbc/psammead-media-player';
 
-const Container = ({ src, title, portrait, showPlaceholder, placeholderSrc }) => (
+const Container = ({ 
+  src, 
+  skin,
+  title, 
+  service,
+  portrait, 
+  showPlaceholder, 
+  placeholderSrc, 
+  placeholderSrcset, 
+  mediaInfo }) => (
```

Comment:
Thanks for updating this. If we're following our Prettier style, the last prop should have a return followed by a comma.
```
  ...
  mediaInfo,
}) => (
  <CanonicalMediaPlayer
  ...
```